thirtyinch-lint/src/main/kotlin/net/grandcentrix/thirtyinch/lint/detector/MissingViewInThirtyInchDetector.kt

```diff
@@ -0,0 +1,57 @@
+package net.grandcentrix.thirtyinch.lint.detector
+
+import com.android.tools.lint.detector.api.Issue
+import com.android.tools.lint.detector.api.JavaContext
+import com.intellij.psi.PsiClass
+import com.intellij.psi.PsiClassType
+import com.intellij.psi.PsiType
+import net.grandcentrix.thirtyinch.lint.TiIssue.MissingView
+import org.jetbrains.uast.UClass
+
+private const val TI_VIEW_FQ = "net.grandcentrix.thirtyinch.TiView"
+private const val PROVIDE_VIEW_METHOD = "provideView"
+private val TI_CLASS_NAMES = listOf(
+        "net.grandcentrix.thirtyinch.TiActivity",
+        "net.grandcentrix.thirtyinch.TiFragment"
+)
+
+class MissingViewInThirtyInchDetector : BaseMissingViewDetector() {
+    companion object {
+        val ISSUE = MissingView.asLintIssue(
+                MissingViewInThirtyInchDetector::class.java,
+                "When using ThirtyInch, a class extending TiActivity, TiFragment or CompositeActivity " +
```

Comment:
This issue don't catch the issues for the CompositeActivity.
Please adjust the comment